package.json

```diff
@@ -57,26 +60,22 @@
     "p-queue": "^6.2.1",
     "p-timeout": "^3.2.0",
     "p-times": "^2.1.0",
+    "paramap-it": "^0.1.1",
     "peer-id": "~0.13.5",
     "peer-info": "~0.17.0",
     "promise-to-callback": "^1.0.0",
-    "promisify-es6": "^1.0.3",
     "protons": "^1.0.1",
-    "pull-length-prefixed": "^1.3.3",
-    "pull-stream": "^3.6.14",
     "varint": "^5.0.0",
     "xor-distance": "^2.0.0"
   },
   "devDependencies": {
     "aegir": "^20.4.1",
+    "async-iterator-all": "^1.0.0",
     "chai": "^4.2.0",
     "datastore-level": "~0.12.1",
     "delay": "^4.3.0",
     "dirty-chai": "^2.0.1",
-    "interface-connection": "~0.3.3",
-    "libp2p-mplex": "~0.8.5",
-    "libp2p-switch": "~0.42.7",
-    "libp2p-tcp": "~0.13.0",
+    "it-pair": "^1.0.0",
```

Comment:
I will create an issue to track the removal of `peer-book` for tests, as well as handle the `peer-store` operations currently done here.

Reply:
we still use `async/queue` for the query `WorkerQueue`. I will create an issue to track that